heka/_common.sls

```diff
@@ -34,6 +34,11 @@ heka_acl_log:
 heka_service:
   service.dead:
   - name: heka
+  - enable: false
+  - sig: 'heka.*/etc/heka'
+
+/etc/init.d/heka:
```

Comment:
The above does not work at all.

I spent more time on the problem, and the way to properly fix it is to kill any heka process with a command line including `hekad -config=/etc/heka` and remove the `/etc/init.d/heka` file to fully disable the `heka` service created by the Heka package.

I updated the patch. Please review one more time.